free-1.md

```diff
@@ -1,54 +1,76 @@
 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea satisfies the need of organization when composing music. We will be helping composers of any skill level organize their compositions. The main reason why I'd like to build a language for organizing composition structures is because a current person's experiences in composing is done on paper or in a language not built for organizing composition structures. There are a lot (like, really, a lot) of langauges that focus on creating music and aiding musicians. But I haven't come across one solely dedicated to organization yet. I hope that the language I am thinking of helps people efficiently organize their thoughts in compositions and frees them to think of other aspects of composing music such as harmony, rhythm, and textures. From here, the hope is that the less time spent thinking about organization leads to more time/efficiency on the other aspects.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+One reason why I see a DSL as a potential solution is because current music software includes a lot of other things. Many programming languages, libraries, and APIs that implement ways to create music focus on making it possible to make cool sounds and patterns. But instead of having that affect one's compositions (following the Sapir-Whorf hypothesis), having a programming langauge specific to enhancing one's compositional creativity would decrease the impact of languages on musician's compositions. A DSL can be used as a preliminary tool or a tool in conjunction with other music software. As stated above, the end goal is to let users easily think of problems other than the structure of a composition that arise when composing.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I am interested in music in different ways. I take piano lessons, do chamber music, play gamelan, play jazz, and am a part of the Pomona College Orchestra. So music (at least classical and jazz) has been a large part of my life, starting when I was fairly young.
+
+I took an independent study in composition with Prof. Alves two years ago, and I learned a couple of things. One is that all pieces, regardless of their genre, always had a plan behind them. Regardless of whether randomness is involved, I believe that almost all music depends on structure and organization. The structure doesn't have to be rigid. For example, a structure can be just 3 sections that have completely random notes, rhythms, and dynamics. However, humans tend to enjoy music they understand, and having structure helps them understand the music.
```

Comment:
Not much of a musician so this question may be completely off, but why would it be better to have this organization done on a computer/DSL as opposed to just organizing it on paper? I could see this being useful for longer pieces of music where it might be hard to fit see everything all at once. But what would be the overall advantages of using this DSL vs. a handwritten plan?